docroot/modules/custom/foia_webform/src/Plugin/QueueWorker/FoiaFormQueue.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Drupal\foia_webform\Plugin\QueueWorker;
+
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Queue\QueueWorkerBase;
+use Drupal\webform\WebformSubmissionInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides base functionality for the FoiaForm Queue Workers.
+ */
+abstract class FoiaFormQueue extends QueueWorkerBase implements ContainerFactoryPluginInterface {
```

Comment:
Maybe we should give the worker a similarly formatted name? e.g. `FoiaSubmissionQueueWorker`